mlir/include/mlir/Dialect/XeGPU/TransformOps/XeGPUTransformOps.td

```diff
@@ -78,4 +78,41 @@ def SetDescLayoutOp : Op<Transform_Dialect, "xegpu.set_desc_layout", [
   }];
 }
 
+def SetGPULaunchThreadsOp
+    : Op<Transform_Dialect, "xegpu.set_gpu_launch_threads", [
+      DeclareOpInterfaceMethods<MemoryEffectsOpInterface>,
+      TransformOpInterface
+    ]> {
+
+  let summary = "Set number of threads for a given gpu.launch operation";
+  let description = "Set number of threads for a given `gpu.launch` operation.";
```

Comment:
Nit: I think if the only useful thing to say in the description is a copy of the summary, the description can just go.

Might there be something more useful to say?